modules/microshift-adding-repos-to-image-builder.adoc

```diff
@@ -11,15 +11,69 @@ Use the following procedure to add the {product-title} repositories to Image Bui
 .Prerequisites
 * Your build host meets the Image Builder system requirements.
 * You have installed and set up Image Builder and the `composer-cli` tool.
-* You have root user access to your build host.
+* You have root-user access to your build host.
 
 .Procedure
 
-* Enable the {product-title} RPM repositories on the build host by running the following command:
+. Create an Image Builder configuration file for adding the `rhocp-4.13` RPM repository source required to pull {product-title} RPMs by running the following command:
 +
-[source,terminal,subs="attributes+"]
+[source,terminal]
 ----
-$ sudo subscription-manager repos \
-    --enable rhocp-{ocp-version}-for-{rhel-major}-$(uname -i)-rpms \
-    --enable fast-datapath-for-{rhel-major}-$(uname -i)-rpms
+$ cat > rhocp-4.13.toml <<EOF
+id = "rhocp-4.13"
+name = "Red Hat OpenShift Container Platform 4.13 for RHEL 9"
+type = "yum-baseurl"
+url = "https://cdn.redhat.com/content/dist/layered/rhel9/$(uname -m)/rhocp/4.13/os"
+check_gpg = true
+check_ssl = false
+system = false
+rhsm = true
+EOF
 ----
+
+. Create an Image Builder configuration file for adding the `fast-datapath` RPM repository by running the following command:
++
+[source,terminal]
+----
+$ cat > fast-datapath.toml <<EOF
+id = "fast-datapath"
+name = "Fast Datapath for RHEL 9"
+type = "yum-baseurl"
+url = "https://cdn.redhat.com/content/dist/layered/rhel9/$(uname -m)/fast-datapath/os"
+check_gpg = true
+check_ssl = false
```

Comment:
```suggestion
check_ssl = true
```

Reply:
@dhellmann With @dhensel-rh 's lgtm, I think these have been QE'd as-is. Could you confirm, @dhensel-rh?

Reply:
I will check the steps again with this change

Reply:
I've checked the _check_ssl = true_ change and it looks ok